kayenta-core/src/main/java/com/netflix/kayenta/metrics/MetricsService.java

```diff
@@ -38,4 +38,9 @@
   default List<Map> getMetadata(String metricsAccountName, String filter) throws IOException {
     return Collections.emptyList();
   }
+
+  List<String> getLocations(String metricsAccountName);
```

Comment:
Could use default methods for both of these so you won't need to touch any of the implementing classes other than `AtlasMetricsService`.

Reply:
Like `getMetadata()` just above.

Reply:
Will do.